arangod/IResearch/IResearchLinkHelper.cpp

```diff
@@ -56,7 +56,7 @@
 namespace {
 
 #ifdef USE_ENTERPRISE
-bool isIgnoredHiddenEnterpriseCollection(std::string const& cName) {
+bool isIgnoredHiddenEnterpriseCollection(std::string_view cName) {
```

Comment:
the call site (in the same file) still calls with an std::string.

Reply:
True